marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/ModelController.java

```diff
@@ -253,4 +268,9 @@ private ModelsService newService() {
     public static class UpdateModelInfoInput {
         public String description;
     }
+
+    public static class ModelReferencesInfo {
+        public List<String> steps;
```

Comment:
A "step" is a JSON object. Looking at the impl below, it looks like these are step names, with one exception - mapping names can be in here as well (sigh). To account for this, I think it's best to be verbose here and call this "stepAndMappingNames", because the list can contain a combination of step and mapping names. That's not pretty, but it captures the reality.